Makefile

```diff
@@ -284,6 +284,33 @@ test-cluster:
 get-cluster-credentials: set-azure-account
 	az aks get-credentials --overwrite-existing -g ${CLUSTER_RESOURCE_GROUP_NAME} -n ${CLUSTER_NAME}
 
+define get_app_first_pod
```

Comment:
I think this is not required as you can do: `kubectl exec -ti deployment/<deployment_name>` and it will automatically take the first pod

Reply:
So much better. Thanks!